heron/spi/src/java/BUILD

```diff
@@ -146,3 +146,19 @@ java_library(
     ),
     deps = statemgr_deps_files,
 )
+
+java_binary(
+    name = "spi-unshaded",
+    srcs = glob([
+        "**/spi/utils/**/*.java",
+        "**/spi/common/**/*.java",
+    ]),
+    deps = utils_deps_files,
+)
+
+genrule(
+    name = "heron-spi-jar",
+    srcs = [":spi-unshaded_deploy.jar"],
+    outs = ["heron-spi.jar"],
+    cmd  = "cp $< $@",
+)
```

Comment:
This might not be enough - we might have to package this jar along with storm and api jars as well - so that we can publish to maven and goes into the install scripts.